modules/programs/wezterm.nix

```
@@ -44,13 +45,45 @@ in {
         how to configure.
       '';
     };
+
+    colorschemes = mkOption {
```

Since "color scheme" is two separate words, I think
```suggestion
    colorSchemes = mkOption {
```
would be more suitable.